lib/lite_config.rb

```diff
@@ -62,13 +67,19 @@ def config_path
   end
 
   def config_filename(name)
-    File.join(config_path, name.to_s + '.yml')
+    name = File.join(config_path, name.to_s + '.yml')
+    name << '.erb' if erb?(name)
+    name
   end
 
   def local_config_filename(name)
     config_filename(name).gsub(/.yml$/, '_local.yml')
   end
 
+  def erb?(name)
+    File.exist?(name + '.erb') || File.extname(name) == '.erb'
```

Comment:
Is there any scenario where the extension of the name would be `.erb`? Usually name is just like `:aws_credentials`